composer.json

```diff
@@ -37,6 +37,7 @@
         "brackets/advanced-logger": "^1.0",
         "brackets/translatable": "^1.0",
         "brackets/media": "^3.0",
+        "brackets/admin-sortable": "^1.0",
```

Comment:
Should we add this package? It requires some work I think, but maybe it is enough.